src/prefect/orion/orchestration/core_policy.py

```diff
@@ -429,6 +440,106 @@ async def before_transition(
             )
 
 
+class HandlePausingFlows(BaseOrchestrationRule):
+    """
+    Governs runs attempting to enter a Paused state
+    """
+
+    FROM_STATES = ALL_ORCHESTRATION_STATES
+    TO_STATES = [states.StateType.PAUSED]
+
+    async def before_transition(
+        self,
+        initial_state: Optional[states.State],
+        proposed_state: Optional[states.State],
+        context: TaskOrchestrationContext,
+    ) -> None:
+        if initial_state is None or not initial_state.is_running():
+            await self.abort_transition(
+                "Cannot pause flows that are not currently running."
+            )
```

Comment:
I think all of these should be REJECT not ABORT as we are not requesting that the client process to _exit_ just telling them they cannot make the requested transition.

Reply:
I'll mirror my comment above, since pausing isn't in the "primary orchestration flow" and not something the engine receives, I do think the "pauser" or "resumer" should be get an abort signal back, there's nothing else to do